_pages/docs/match-results.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: inner
+title: Match Results
+lead_text: ''
+permalink: /docs/match-results/
+---
+
+# Match Results
+
```

Comment:
I think it would be worthwhile just putting the function signature in here.  just having the return payload makes me fill in a cognitive gap that was tricky